### PR TITLE
Update structural_bioinformatics.md

### PR DESCRIPTION
**Pull request by csimong**

### CODE REVIEW

File: pages/your_domain/structural_bioinformatics.md [hunk 1]
```diff
@@ -48,3 +48,19 @@ Researchers in the field should be able to find predictions of macromolecular st
...
+* As well as for computationally solved structures, model coordinates and metadata for experimentally solved structures are also preferably stored using the standard PDB archive format {% tool "pdbx-mmcif-format-and-tools" %} and the {% tool "pdbx-mmcif-modelcif-extension-dictionary" %}, respectively.
+* Data model and metadata standards for submitting data underpinning macromolecular models depend on the experimental method used. EMDB map distribution format description has broadly followed [CCP4](https://www.ccp4.ac.uk/html/maplib.html) and [MRC](https://www2.mrc-lmb.cam.ac.uk/research/locally-developed-software/image-processing-software/) map format. Metadata is contained in a header file, an XML file that follows the XSD data model. EMPIAR data model schema consists of the main empiar.xsd XML schema file and additional requirements in empiar.sch in Schematron format (see [here](ftp://ftp.ebi.ac.uk/pub/databases/emtest/empiar/schema/) for more details). BMRB (meta)data distribution format is based on NMR-STAR, an extension of the Self-defining Text Archive and Retrieval (STAR) file format.
```
Comment (bedroesb) [inline pages/your_domain/structural_bioinformatics.md:61]: I would not link to an ft[ address since most browsers can't open it ...

**Comment by bedroesb** [comment]
@DanFaria @pinarpink @bianchini88 @lauportell @wittigue I updted this pull request so it makes use of the new way of tagging tools. I think it is ready for review! Can one of you have a look?

File: _data/CONTRIBUTORS.yaml [hunk 2]
```diff
@@ -540,3 +540,5 @@ Dirk Winkelhardt:
     orcid: 0000-0001-8770-2221
     email: dirk.winkelhardt@rub.de
     affiliation: Ruhr-Universität Bochum, Medzinisches Proteom-Center & ELIXIR Germany Forschungszentrum Jülich
+Carolina Simón Guerrero:
```
Comment (bianchini88) [inline _data/CONTRIBUTORS.yaml:543]: If possible, add also the other fields, especially the ORCID.

File: pages/your_domain/structural_bioinformatics.md [hunk 3]
```diff
@@ -48,3 +48,19 @@ Researchers in the field should be able to find predictions of macromolecular st
   * input data needed for the modelling steps. For instance in homology modelling this could include the {% tool "pdb" %} identifiers for the template structures used for modelling and their alignments to the target protein;
   * model quality estimates.
 * If necessary, accompanying data can be provided in separate files using different file formats. The files can be added to ModelArchive depositions and referred to in the PDBx/mmCIF ModelArchive extension format.
+
+## Storing and sharing experimentally solved structures
+ 
+### Description
+
+Experimentally solved atomic structures of molecules can be obtained by several methods, such as X-ray crystallography,  Nuclear Magnetic Resonance (NMR) spectroscopy, and 3D Electron Microscopy. Here you can find useful tools and guides for storing and sharing structure models based on these methods.
+* Structure models resulting from experimental methods are broadly in the {% tool "pdb" %} under the [CC0 1.0 Universal (CC0 1.0) Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/).
```
Comment (bianchini88) [inline pages/your_domain/structural_bioinformatics.md:57]: I think the word "available" is missing after "broadly" (or something similar)

File: pages/your_domain/structural_bioinformatics.md [hunk 4]
```diff
@@ -48,3 +48,19 @@ Researchers in the field should be able to find predictions of macromolecular st
...
+* Additionally, raw and intermediate data associated with the structure model can be also published in different curated data archives depending on the methods used. Raw EM data and processed 3D volumes and tomograms can be stored in {% tool "empiar" %} and {% tool "emdb" %}, respectively. Raw data from NMR studies can be stored in {% tool "bmrb" %}. For X-ray diffraction experiments raw data can be stored in {% tool "irrmc" %} and {% tool "sbgrid-data-bank" %}. Data submitted in these repositories can be cross-referenced to related PDB entries.
```
Comment (bianchini88) [inline pages/your_domain/structural_bioinformatics.md:58]: >Data submitted in these repositories

Probably "to" is the right preposition here. Alternative, "Data stored in..."

File: pages/your_domain/structural_bioinformatics.md [hunk 5]
```diff
@@ -48,3 +48,19 @@ Researchers in the field should be able to find predictions of macromolecular st
...
+* To extend molecular context, structure models can be visualized and analyzed along with respective volume maps and {% tool "uniprot" %} sequences using data aggregators such as {% tool "3dbionotes" %}, either if they have been published in PDB and EMDB or if not. A bunch of biochemical, biomedical and validation annotations would be mapped on the structure model coordinates and sequence to let the user better understand macromolecular function and interactions. Users can also use the [COVID-19 Structural Hub](https://3dbionotes.cnb.csic.es/ws/covid19), a dedicated summary view for all published SARS-CoV-2 structure models.
```
Comment (bianchini88) [inline pages/your_domain/structural_bioinformatics.md:59]: As we are adhering to the UK spelling, verbs ending in "ized" (e.g. visualized and analyzed) should end in "ised" instead. This is mentioned here: https://rdmkit.elixir-europe.org/style_guide

Comment (bianchini88) [inline pages/your_domain/structural_bioinformatics.md:59]: "A bunch of" This is very colloquial. Consider using words such as "A collection", "A selection", "A number"

File: pages/your_domain/structural_bioinformatics.md [hunk 6]
```diff
@@ -48,3 +48,19 @@ Researchers in the field should be able to find predictions of macromolecular st
...
+* [ARIA](https://aria.services)  is a platform that projects and infrastructures can use to manage Access, from proposal submission through to reporting. It provides tools for facilities within a distributed infrastructure to manage their equipment.
+
+  If using ARIA to manage access to the facility, it will shortly be possible to link output data and metadata with proposals, publications and other outputs. This work is being funded by a number of EU funded grants.
```
Comment (bianchini88) [inline pages/your_domain/structural_bioinformatics.md:65]: Why not add this feature to the previous paragraph? The sentence regarding funders is probably not useful for the reader (and I suppose all the tools mentioned on every page are funded somehow, and we do not want to write a similar sentence for each one).

**Review by bianchini88 — CHANGES_REQUESTED** [review]

File: _data/CONTRIBUTORS.yaml (hunk 2 above)
Comment (csimong) [reply _data/CONTRIBUTORS.yaml:543]: Done! :smile:

File: _data/CONTRIBUTORS.yaml [hunk 7]
```diff
@@ -540,6 +540,12 @@ Dirk Winkelhardt:
...
+    git: csimong
+    email: carolina.simon@cnb.csic.es
+    orcid: 0000-0002-0735-4317
+    role: editor
```
Comment (bianchini88) [inline _data/CONTRIBUTORS.yaml:547]: @csimong I do not think you have the "role" of an "editor" (although you can become one if you want to).
I would suggest getting rid of that line for now.

Comment (csimong) [reply _data/CONTRIBUTORS.yaml:547]: No problem! I will delete it.

**Comment by csimong** [comment]
> @csimong I know it has been a while, and we did not forget your contribution! Since the contentathon we have been working on a new way of adding tools to the RDMkit, as described [here](https://rdmkit.elixir-europe.org/tool_resource_update).
> 
> Would it be possible to update your text by linking towards the tools using the `{% tool "tool-id" %} snippet? The tools that are not yet in our tool list can be added to your pull request by editing this [YAML file](https://github.com/csimong/rdmkit/blob/patch-2/_data/tool_and_resource_list.yml).
> 
> If you need any help, please reach out!

Hi @bedroesb ! 
I can see that you have done almost all the work to update the text with the new way of tagging the tools. Thank you so much for your work!
I have also added some lines for those tools that remained to be included.

Hope it is useful :smile:

File: pages/your_domain/structural_bioinformatics.md (hunk 4 above)
Comment (csimong) [reply pages/your_domain/structural_bioinformatics.md:58]: Done!

File: pages/your_domain/structural_bioinformatics.md (hunk 6 above)
Comment (csimong) [reply pages/your_domain/structural_bioinformatics.md:65]: Done!

File: pages/your_domain/structural_bioinformatics.md (hunk 1 above)
Comment (csimong) [reply pages/your_domain/structural_bioinformatics.md]: I have replace the ftp link for other related https link they provide in the paper (https://ftp.ebi.ac.uk/pub/databases/emtest/empiar/schema/). Would it be better this way?

File: pages/your_domain/structural_bioinformatics.md (hunk 5 above)
Comment (csimong) [reply pages/your_domain/structural_bioinformatics.md]: Done!

Comment (csimong) [reply pages/your_domain/structural_bioinformatics.md]: Done!

**Pull request merged**